CHANGELOG.md

```diff
@@ -7,13 +7,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-## [0.1.5] - 2023-10-10
+## [0.1.7] - 2023-10-10
```

Comment:
This PR should be under [Unreleased]. We haven't released 0.1.7 yet, so don't change what was under 0.1.5.

Reply:
Sorry, brain fart.